examples/detached_app/src/main.rs

```diff
@@ -0,0 +1,91 @@
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")] // hide console window on Windows in release
+
+use std::time::Instant;
+
+use eframe::{egui, DetachedResult};
+use winit::{
+    event::{ElementState, Event, WindowEvent},
+    event_loop::ControlFlow,
+};
+
+fn main() {
+    env_logger::init(); // Log to stderr (if you run with `RUST_LOG=debug`).
+    let options = eframe::NativeOptions {
+        initial_window_size: Some(egui::vec2(320.0, 240.0)),
+        ..Default::default()
+    };
+    let event_loop = eframe::EventLoopBuilder::<eframe::UserEvent>::with_user_event().build();
+
+    // A detached window managed by eframe
+    let mut detached_app = eframe::run_detached_native(
+        "My egui App",
+        &event_loop,
+        options,
+        Box::new(|_cc| Box::<MyApp>::default()),
+    );
+    // Winit window managed by the application
+    let winit_window = winit::window::WindowBuilder::new()
+        .build(&event_loop)
```

Comment:
Giving this a title of e.g. "winit window" would help explain to the user what is going on